test/integration/xpack-stig.yml

```diff
@@ -3,7 +3,7 @@
 # Make sure to change the variables below in the vars section
 # and in the kitchen.vagrant.yml file
 
-- import_playbook: roles/elasticsearch/test/integration/xpack.yml
+- import_playbook: elasticsearch/test/integration/xpack.yml
```

Comment:
Hmm, still not working for me. 

```
 ERROR! the role 'elasticsearch' was not found in /tmp/kitchen/elasticsearch/test/integration/roles:/tmp/kitchen/roles:/tmp/kitchen/elasticsearch/test/integration

       The error appears to have been in '/tmp/kitchen/elasticsearch/test/integration/xpack.yml': line 7, column 7, but may
       be elsewhere in the file depending on the exact syntax problem.
```

Is there anything else I need to setup to get this working? This is what I'm currently doing.

```
bundle install
export KITCHEN_YAML=.kitchen.vagrant.yml
export ES_XPACK_LICENSE_FILE="$(pwd)/license.json"
bundle exec kitchen converge xpack-stig-centos-7
```

Reply:
Hey @Crazybus, returning to this sorry about the long long delay. I got a new PC in between this time so I was trying to set this up fresh. It looks like what I ran into is that kitchen basically dumps the entire directry into `/tmp/kitchen` on the target machine. So either the directory `ansible-elasticsearch` needs to be renamed to just `elasticsearch` or the role needs to be changed to match the directory `ansible-elasticsearch`. Even if I try to just run any of the other xpack converges I run into that same error as you did. 

I just renamed the directory to elasticsearch and it starts to run the plays. Does that work for you? This way kitchen dumps the directory into the target as `elasticsearch` and then it can find it fine in `/tmp/kitchen/roles/elasticsearch` when looking at the `{ role: elasticsearch }`. I'm not even sure how I got around this now on my other machine.